QNX change from QNX's aports (extra/xdg-utils/busybox-sed.patch).

Taken from qnx-ports/aports extra/xdg-utils -- the APKBUILD the QNX Everywhere 8.0.4r1
image's package is built from.

Upstream-Status: QNX-specific

Busybox sed allows } to be a label so we need to add a ;.

Ref https://gitlab.freedesktop.org/xdg/xdg-utils/-/merge_requests/126

--- a/scripts/xdg-screensaver.in
+++ b/scripts/xdg-screensaver.in
@@ -363,7 +363,7 @@ screensaver_kde3()
 
 xset_screensaver_timeout()
 {
-  xset q | sed '/^Screen Saver:/,/^[^ ]/ { s/.*timeout: *\([0-9]*\).*/\1/; t }; d'
+  xset q | sed '/^Screen Saver:/,/^[^ ]/ { s/.*timeout: *\([0-9]*\).*/\1/; t; }; d'
 }
 
 screensaver_xserver()

